test/diamond-e2e/testdata/cycle-backward.yaml

```diff
@@ -5,44 +5,28 @@ metadata:
 spec:
   vertices:
     - name: in
-      containerTemplate:
-        env:
-          - name: NUMAFLOW_RUNTIME
-            value: "rust"
       source:
         http: {}
     - name: cat
-      containerTemplate:
-        env:
-          - name: NUMAFLOW_RUNTIME
-            value: "rust"
       udf:
         container:
           image: quay.io/numaio/numaflow-go/map-cat:stable # A UDF which simply cats the message
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Why update them?

Reply:
To avoid downloading some images again and again, I feel like the github action has already reached the limit.

Reply:
e2e tests are more stable now